libnetwork/network.go

```diff
@@ -579,7 +577,6 @@ func (n *Network) MarshalJSON() ([]byte, error) {
 	netMap["ingress"] = n.ingress
 	netMap["configOnly"] = n.configOnly
 	netMap["configFrom"] = n.configFrom
-	netMap["loadBalancerIP"] = n.loadBalancerIP
```

Comment:
There's no need to keep this property marshaled into the on-disk state to preserve the ability to downgrade, since `overlay` networks are never persisted in the first place.